engine/src/test/java/org/camunda/bpm/engine/test/bpmn/el/ExpressionManagerTest.java

```diff
@@ -108,4 +113,32 @@ public void testSetVariableByExpressionFromListener() {
     // then
     assertEquals(1L, historyService.createHistoricVariableInstanceQuery().variableValueEquals("myCounter", 6).count());
   }
+
+  @Test
+  @RequiredHistoryLevel(ProcessEngineConfiguration.HISTORY_FULL)
+  public void testJuelExpressionWithNonPublicClass() {
+    final BpmnModelInstance process = Bpmn.createExecutableProcess("testProcess")
+        .startEvent()
+        .exclusiveGateway()
+        .condition("true", "${list.contains('foo')}")
+        .userTask("userTask")
+        .moveToLastGateway()
+        .condition("false", "${!list.contains('foo')}")
+        .endEvent()
+        .done();
```

Comment:
```suggestion
        .startEvent()
        .exclusiveGateway()
          .condition("true", "${list.contains('foo')}")
          .userTask("userTask")
        .moveToLastGateway()
          .condition("false", "${!list.contains('foo')}")
          .endEvent()
        .done();
```
Isn't the BPMN fluent builder definition easier to read if some indention is used?